explorer/src/explorer/ExplorerOptions.hs

```diff
@@ -136,6 +136,8 @@ argsParser = do
         long "static-peers" <>
         help "Don't use Kademlia, use only static peers"
 
+    sscAlgo <- CLI.sscAlgoOption
```

Comment:
It should be removed. It produces a warning which wasn't caught by CI for unknown reason (I asked in #sl-devops about it).